src/bitcoind.cpp

```diff
@@ -143,7 +143,7 @@ static bool AppInit(int argc, char* argv[])
 #pragma GCC diagnostic push
 #pragma GCC diagnostic ignored "-Wdeprecated-declarations"
 #endif
-            tfm::format(std::cout, PACKAGE_NAME " daemon starting\n");
+            tfm::format(std::cout, PACKAGE_NAME " starting\n");
 
             // Daemonize
             if (daemon(1, 0)) { // don't chdir (1), do close FDs (0)
```

Comment:
Why are you removing `daemon` here?

Reply:
> Why are you removing `daemon` here?

https://github.com/bitcoin/bitcoin/pull/16489#discussion_r309873325 combined with the goal of harmonizing the various messages, thus my explanation here https://github.com/bitcoin/bitcoin/pull/16489#issuecomment-517408355.

Reply:
I updated the PR description to explain the motivation.